### PR TITLE
Component cleanup

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: app/components/ImageField/ImageField.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ class ImageField extends Component {
   }
 
   render() {
-    const { field, label } = this.props;
+    const field = this.props.field || {}; // TODO: test.
```
Comment (MarcelCutts) [inline app/components/ImageField/ImageField.js]: Is this TODO done?

File: app/components/Preview/Preview.js [hunk 2]
```diff
@@ -5,31 +5,26 @@ import Meetup from '../Meetup/Meetup';
 import SpeakerPreview from '../SpeakerPreview/SpeakerPreview';
 import SponsorPreview from '../SponsorPreview/SponsorPreview';
 
-const Preview = ({ text }) => (
+function getIn(haystack, key) {
+  return haystack && haystack[key] || [];
```
Comment (MarcelCutts) [inline app/components/Preview/Preview.js]: Not sure about the `haystack` naming here

File: app/components/ImageField/ImageField.js (hunk 1 above)
Comment (lpil) [inline app/components/ImageField/ImageField.js]: Nope. Figure I'd do this in a later PR as this one has got large

File: app/components/Preview/Preview.js (hunk 2 above)
Comment (lpil) [inline app/components/Preview/Preview.js]: `object` ?

File: app/containers/Preview.js [hunk 3]
```diff
@@ -1,18 +1,12 @@
 import { connect } from 'react-redux';
 import PreviewComponent from '../components/Preview/Preview.js';
 
-const getEditorContent = (state) => {
-  if (state && state.form && state.form.editor && state.form.editor.values) {
+const mapStateToProps = (state) => {
+  try {
     return { ...state.form.editor.values };
+  } catch (_) {
```
Comment (MarcelCutts) [inline app/containers/Preview.js]: Underscore is a blank match in some languages but here it's just a variable

Comment (lpil) [inline app/containers/Preview.js]: Yup, just stating that I'm not interested in using this variable. Would you prefer it renamed?

**Pull request merged**